docs/data/material/varnish/header/header.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Varnish Header
+githubLabel: 'varnish: header'
+---
+
+# Header
+
+<p class="description">Official AI2 Header.</p>
+
+
+## When To Use
+
+You should put this at the top of the page. It is highly recommended to be present on every page of
+your application.
+
+### API
+
+| Property      | Description                                                         | Type    | Default |
+| ------------- | ------------------------------------------------------------------- | ------- | ------- |
+| bannerAlwaysVisible | Prevents the AI2 banner from disappearing when the page is scrolled | boolean | false   |
+| customBanner | A Custom AI2 banner that you can inject into the header via this prop | JSX / TSX Element; if left undefined, the banner is the default AI2 banner | undefined |
+| scrollTriggerTarget | If you choose to use the AI2 Header with Smart Banner within a div, you can change the scroll target to be the specific div it is in. Note: This should rarely be used, reach out to the team if you find yourself using this often. | HTMLDivElement | undefined |
+
+## Sub Components
+
+### Header.Columns API
```

Comment:
please add an api section (in the correct spot) for the new collapse attribute

Reply:
the Collapse component is not external-facing so I don't think it makes sense to have it on the Header docs. The only way to manipulate it is through the scrollTriggerTarget prop which has already been listed in the API section; let me know if you'd like to chat about this further though.

Reply:
oh, that makes sense!